src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -414,6 +417,10 @@ export class HaSceneEditor extends SubscribeMixin(
     `;
   }
 
+  protected firstUpdated() {
+    this.registerShortcut(HAKeyboardShortcut.CTRL_S, () => this._saveScene());
+  }
```

Comment:
This works, but it breaks if I use the standard shorthand for void callbacks:

```
protected firstUpdated() {
    this.registerShortcut(HAKeyboardShortcut.CTRL_S, this._saveScene); // <-- saveScene fails from corrupted "this"
}
```

By the time we get into `_saveScene` method, the `this` has been corrupted and doesn't have the needed properties. 

If anyone can help explain why I'd be happy to find a solution.

Reply:
Your current code is fine. Arrow functions keep the context, normal functions create a new context.